sw_i2c/sensirion_sw_i2c.c

```diff
@@ -37,12 +37,13 @@
 #define DELAY_USEC (SENSIRION_I2C_CLOCK_PERIOD_USEC / 2)
 
 static int8_t sensirion_wait_while_clock_stretching(void) {
-    uint8_t timeout = 100;
+    /* Maximal timeout of 150ms (SCD30) in sleep polling cycles */
+    uint32_t timeout_cycles = 150000 / SENSIRION_I2C_CLOCK_PERIOD_USEC;
```

Comment:
Shouldn't this be configurable by the caller?

Reply:
@rnestler In the ideal-world case (error-free operation) it won't matter - feel free to amend. I didn't think of that possibility as long as it covered the worst case - but you're definitely right - we don't know the worst case. What do you think of adding this at the top?
```
#ifndef SENSIRION_I2C_CLOCK_STRETCHING_TIMEOUT_USEC
/**
 * Configure the waiting for clock to be released timeout by an i2c slave.
 * The value must be at least SENSIRION_I2C_CLOCK_PERIOD_USEC.
 */
#define SENSIRION_I2C_CLOCK_STRETCHING_TIMEOUT_USEC 150000
#endif /* SENSIRION_I2C_CLOCK_STRETCHING_TIMEOUT_USEC */
```